README.md

```diff
@@ -156,6 +156,29 @@ The simplest way to do this is to create a `.npmrc` file in the root of your pro
 public-hoist-pattern[]=*
 ```
 
+## Running the tests
+
+To run the tests, ensure that the dependencies are installed as follows:
+
+```shell
+npm install
+```
+
+Then run the tests:
+
+```shell
+npm test
+```
+
+In order to run the end-to-end (E2E) tests, you'll need to set an environment variable `NETLIFY_AUTH_TOKEN` to a valid Netlify personal access token. This can be obtained from the [Netlify UI](https://docs.netlify.com/cli/get-started/#obtain-a-token-in-the-netlify-ui).
+
+Then run the E2E tests:
+
+```shell
+NETLIFY_AUTH_TOKEN=your-token-here npm test:next
```

Comment:
This might change anyway, based on @ascorbic's comment about passing in a site ID, but you missed the 'run' from `npm run test:next`

Reply:
In this case the 'run' isn't necessary because `npm` aliases it - https://docs.npmjs.com/cli/v8/commands/npm-test

Reply:
That works for `npm test`, but not `npm test:next` etc

Reply:
For my reference, [Matt's comment about site ID](https://netlify.slack.com/archives/C01TKAEBP3Q/p1671190416692539?thread_ts=1671182790.434209&cid=C01TKAEBP3Q)

> You can pass in a site ID as an env var for the tests